### PR TITLE
Odim 6889: Add process logs for svc-event

### PR DESCRIPTION
**Pull request by chauberahul1993**

### CODE REVIEW

**Review by arahimanshaik — APPROVED** [review]

File: plugin-lenovo/main.go [hunk 1]
```diff
@@ -67,7 +68,11 @@ func main() {
 
 	// RunReadWorkers will create a worker pool for doing a specific task
 	// which is passed to it as Publish method after reading the data from the channel.
-	go common.RunReadWorkers(lphandler.Out, lpmessagebus.Publish, 5)
+
+	// should be removed when context from svc-api is passed to this function
+	ctx := context.TODO()
+
+	go common.RunReadWorkers(ctx, lphandler.Out, lpmessagebus.Publish, 5)
```
Comment (hemkapoor) [inline plugin-lenovo/main.go:75]: we should increment thread ID according to the go routines.

File: svc-events/evcommon/common.go [hunk 2]
```diff
@@ -142,15 +143,15 @@ var EMBTopics EmbTopic
...
 	for {
 		pluginList, err := evmodel.GetAllPlugins()
 		if err != nil {
-			l.Log.Error(err.Error())
+			l.LogWithFields(ctx).Error(err.Error())
 			return
 		}
 		for i := 0; i < len(pluginList); i++ {
-			go st.getPluginStatus(context.TODO(), pluginList[i]) //TODO: Pass context
+			go st.getPluginStatus(ctx, pluginList[i])
```
Comment (hemkapoor) [inline svc-events/evcommon/common.go:154]: same..as above

File: svc-events/evcommon/common.go [hunk 3]
```diff
@@ -534,19 +535,22 @@ func ProcessCtrlMsg(data interface{}) bool {
...
+	ctx = context.WithValue(ctx, common.ActionName, "SubscribePluginEMB")
 	pluginList, err := GetAllPluginsFunc()
 	if err != nil {
-		l.Log.Error(err.Error())
+		l.LogWithFields(ctx).Error(err.Error())
 		return
 	}
 	for i := 0; i < len(pluginList); i++ {
-		go st.getPluginEMB(pluginList[i])
+		go st.getPluginEMB(ctx, pluginList[i])
```
Comment (hemkapoor) [inline svc-events/evcommon/common.go:549]: same as above

File: svc-events/events/subscriptionCache.go [hunk 4]
```diff
@@ -33,17 +34,17 @@ var (
...
-	getAllSubscriptions()
-	getAllAggregates()
-	getAllDeviceSubscriptions()
-	go initializeDbObserver()
+func LoadSubscriptionData(ctx context.Context) {
+	l.LogWithFields(ctx).Debug("Event cache is initialized")
+	getAllSubscriptions(ctx)
+	getAllAggregates(ctx)
+	getAllDeviceSubscriptions(ctx)
+	go initializeDbObserver(ctx)
```
Comment (hemkapoor) [inline svc-events/events/subscriptionCache.go:42]: increment thread ID

**Review by sriteja-sabbineni — APPROVED** [review]

**Review by hemkapoor — APPROVED** [review]

**Review by amar-shalgar — APPROVED** [review]

**Pull request merged**